app/assets/stylesheets/styles/components/pages/_home.css.scss

```diff
@@ -41,7 +41,7 @@ video { display: block; }
   bottom: 0;
   left: 0;
   right: 0;
-  background-color: rgba(0,0,0,0.35);
+  background-color: rgba(0,0,0,0.25);
```

Comment:
`0.25` should be written without a leading zero as `.25`<br>Commas in function arguments should be followed by a single space